app/Http/Controllers/AppealController.php

```diff
@@ -112,15 +113,47 @@ public function appeallist()
                 });
         }
 
-        $hiddenStatuses = $isDeveloper
-            ? [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_INVALID]
-            : [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND, Appeal::STATUS_INVALID];
+        $appealtypes = ['assigned'=>'Assigned to me','unassigned'=>'All unreserved open appeals','reserved'=>'Open reserved appeals'];
+        if($isDeveloper) { $appealtypes['developer']='Developer access appeals'; }
```

Comment:
This if can be removed, instead let's not show empty lists on the view

Reply:
I don't know what you refer to by 'this' and I don't see what can be removed. These are table headers.
If we want to not show empty lists, I would have to prequery a whole bunch of information, and people might get worried if no table shows up at all because there are 0 appeals that need handling.